### ISSUE TITLE
problem6 image ordering

### ISSUE THREAD

**Issue by kareemostafa** [chulminy/CIVE497-CIVE700#64]

**Comment by chulminy** [comment]
Yes. You need to use the information. Actually, I think the Matlab tutorial code used the order information

**Comment by kareemostafa** [comment]
This is the best stitching I got so far (I followed the same concept in the tutorial: sift==>homography/ransac==>apply homography to the images). I have some sort of justifications of why they are the way they are. Am I fine submitting this as-is or do I need to work on it to make it look like the one in the tutorial?
![image](https://user-images.githubusercontent.com/35819560/54484152-babea180-4836-11e9-8a73-545d194f6374.png)

**Comment by chulminy** [comment]
You should implement all process in Matlab. I think the problem is that you didn't implement `imwarp` in the python. The hommography is not correct as well.

**Comment by kareemostafa** [comment]
imwarp is implemented and the homography is obtained following MATLAB's procedure and the same way I got it in the previous problems (SIFT/RANSAC) and they work properly there. Still getting the same result as shown above

**Comment by chulminy** [comment]
Um... I think we can ask other students who are working with Python.
Is there any idea?
